llvm/test/SYCLLowerIR/esimd_lower_crash_zext.ll

```diff
@@ -0,0 +1,12 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -LowerESIMD -S < %s | FileCheck %s
+
```

Comment:
please add a description what this test checks

Reply:
Done.

Reply:
I committed slight clarification to speed up things, hope you don't mind.